src/localconf.js

```diff
@@ -43,20 +28,21 @@ function buildConfig(opts) {
   const submissionApiPath = `${hostname}api/v0/submission/`;
   const apiPath = `${hostname}api/`;
   const submissionApiOauthPath = `${hostname}api/v0/oauth2/`;
-  // let credentialOauthPath = `${hostname}middleware/oauth2/v0/`;
   const graphqlPath = `${hostname}api/v0/submission/graphql/`;
   let userapiPath = `${hostname}user/`;
-  let credentialCdisPath = `${userapiPath}credentials/cdis/`;
+  const jobapiPath = `${hostname}/job/`;
+  const credentialCdisPath = `${userapiPath}credentials/cdis/`;
   let login = {
     url: `${userapiPath}login/google?redirect=`,
     title: 'Login from Google',
   };
   const loginPath = `${userapiPath}login/`;
   const graphqlSchemaUrl = `${hostname}data/schema.json`;
-  // let credentialPath = `${hostname}middleware/aws/v0/access_key/`;
 
-  let appname = 'Unasigned';
-  let navItems = [];
+  const topItems = [
+    { icon: 'upload-white', link: '/submission', name: 'Data Submission' },
+    { link: 'https://uc-cdis.github.io/gen3-user-doc/user-guide/guide-overview', name: 'Documentation' },
```

Comment:
should link to bloodpac specific doc site for bpa 
https://github.com/uc-cdis/data-portal/issues/268